ur_bringup/launch/ur10_bringup.launch

```diff
@@ -19,7 +19,7 @@
   <arg name="visual_params" default="$(find ur_description)/config/ur10/visual_parameters.yaml"/>
 
   <!-- robot model -->
-  <include file="$(find ur_description)/launch/ur10_upload.launch">
+  <include file="$(find ur_description)/launch/load_ur10.launch">
```

Comment:
`ur_bringup` will be removed, but in a later PR.

So I had to fix these lines.